scripts/feedme.coffee

```diff
@@ -0,0 +1,118 @@
+# Description:
+#   Shows what"s edible.
+#
+# Notes:
+#   This includes
+#    * fetching from fabrik
+#    * fetching from ernis
+#
+# Author: cholter
+
+urllib = require "urllib-sync"
+jsdom = require "jsdom"
+require "datejs"
+
+DAY = 1000 * 60 * 60  * 24
+
+getDateDiff = (date1, date2, interval) ->
+  return Math.round((date1.getTime() - date2.getTime()) / interval)
+
+fetch = (target) ->
+  return urllib.request(target).data
+
+
+class Fabrik
+  target : "http://www.diefabrik.co.at/mittagsmenue/index.html"
+  holidayMagic : "urlaub"
+  errors:
+    closed : "fabrik is closed"
+    menuFromPast :"the menu is outdated"
+    menuFromFuture :"the menu is from the future"
+    resting :"fabrik is on a day off"
+    holiday :"fabrik is probably on holiday, fall back to manual check"
+
+  constructor: (@robot) ->
+
+  checkHoliday: (text) =>
+    return text.toLowerCase().indexOf(@holidayMagic) > 0
+
+  parseDates: ($) ->
+    docDate = $("h2:eq(0)").html()
+    return /(\d{2}\.\d{2}\.\d{4}) bis (\d{2}\.\d{2}\.\d{4})/.exec(docDate)
+
+  extractMeal: ($, day) ->
+
+    # Saturday & Sunday
+    return @errors.closed if day == 6 or day == 0
+    menu = $(".contenttable .tr-#{(day-1)*2} .td-2").html()
+    # Ruhetag
+    return @errors.resting if menu.toLowerCase() == 'ruhetag'
+    # all good :)
+    return menu
+
+  getMenu: () =>
+    now = new Date()
+    day = now.getDay()
+    lastCheck = new Date(@robot.brain.get "feedme.fabrik.lastCheck" or 0)
```

Comment:
The `or` operator takes precedence over function calls, so this is missing parentheses.